net/paho.mqtt.c/Portfile

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+PortGroup           cmake 1.1
+
+github.setup        eclipse paho.mqtt.c 1.3.9 v
+categories          net
+platforms           darwin
+maintainers         nomaintainer
+license             Eclipse Public License - v-2.0
+
+description         Library to enable applications to connect to an MQTT broker to publish and receive messages.
+
+long_description    MQTT and MQTT-SN are lightweight publish/subscribe \
+                    messaging transports for TCP/IP and connectionless \
+                    protocols (such as UDP) respectively. The Eclipse \
+                    Paho project provides open source, mainly client side, \
+                    implementations of MQTT and MQTT-SN in a variety of \
+                    programming languages.
+
+# We enable SSL so that the paho.mqtt.cpp library builds without issues
+configure.args-append \
+                    -DPAHO_WITH_SSL=ON
```

Comment:
Since we are enabling SSL here, we need to declare OpenSSL as a dependency needed by this port.

These days, we can actually do that with the `openssl` portgroup:

```
PortGroup           cmake 1.1
PortGroup           github 1.0
PortGroup           openssl 1.0
```

Adding that portgroup is enough to automatically add OpenSSL to the dependencies list and configure CMake to build against it.